### PR TITLE
PR UAVCAN Firmware Updating

### PR DESCRIPTION
**Pull request by samchamberlin**
Addresses https://github.com/PX4/PX4-user_guide/issues/1424.

Unsure if we need to keep the VectorControl ESC instructions.

### CODE REVIEW

File: en/uavcan/node_firmware.md [hunk 1]
```diff
@@ -1,128 +1,72 @@
...
-
-### Flashing the UAVCAN Bootloader
+After restarting the flight controller the UAVCAN device will automatically be updated with new firmware.
+If successful the firmware binary will be removed from the root directory and there will be a file named **XX.bin** in the **/ufw** directory of the SD card.
 
-Before updating firmware via UAVCAN, the Pixhawk ESC 1.6 requires the UAVCAN bootloader be flashed.
-To build the bootloader, run:
+:::note
+UAVCAN firmware is usually created as part of the build process.
+Commonly the firmware image is built in **build/_name_of_your_uavcan_device_**,  where _name_of_your_uavcan_device_ is specific to the device you are flashing (e.g. **build/ark_can-flow_default**), and will be named with the format **XX-X.X.XXXXXXXX.uavcan.bin**.
```
Comment (hamishwillee) [inline en/uavcan/node_firmware.md:20]: Is it actually named `XX-X.X.XXXXXXXX.uavcan.bin` or are the X placeholders for versions or something else? Does it matter as long as the extension is correct?

**Comment by hamishwillee** [comment]
Thanks very much @samchamberlin ! I've subedited to make fully separate firmware update and firmware creation. Can you please check to make sure it is still correct (before I merge)?

File: en/uavcan/node_firmware.md (hunk 1 above)
Comment (samchamberlin) [reply en/uavcan/node_firmware.md:20]: X is placeholding for numbers, but to my knowledge nothing else will have the same '.uavcan.bin' extension either.

**Comment by samchamberlin** [comment]
Made a small adjustment for ARK documentation. Looks good now! Thanks @hamishwillee

File: en/uavcan/node_firmware.md (hunk 1 above)
Comment (hamishwillee) [reply en/uavcan/node_firmware.md:20]: Thanks. That's OK, just wanted to make sure that "format" was the right word to use. I think people will infer the extension is required.

**Comment by hamishwillee** [comment]
Looks great. Merged. Thanks for the help!

**Pull request merged**